tests/ssl-opt.sh

```diff
@@ -202,6 +202,7 @@ get_config_value_or_default() {
 
 requires_config_value_at_least() {
     VAL="$( get_config_value_or_default "$1" )"
+    VAL=${VAL//[^0-9]*}
```

Comment:
Please explain in the commit message why you need this change and what test failure it does fix.

Reply:
Done